Assignment 3.Rmd

```diff
@@ -26,13 +36,13 @@ The algorithm will treat each row as a value belonging to a person, so we need t
 
 ```{r}
 
-K2 <- 
+K2 <- K1[,-1]
 
 ```
 
 It is important to think about the meaning of missing values when clustering. We could treat them as having meaning or we could remove those people who have them. Neither option is ideal. What problems do you foresee if we recode or remove these values? Write your answers below:
 
-
+If the missing values are not deleted , the k means model cann't be fitted because kmeans model can not handle missing values.
```

Comment:
Because the deletion of data will decrease the reliability of the sample, missing data is also part of the data itself, if we delete it will equal to reduce the reliability of our data.